tests/ui/deprecation/deprecated_no_stack_check.rs

```diff
@@ -1,4 +1,4 @@
-//@ normalize-stderr: "you are using [0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z0-9]+)?( \([^)]*\))?" -> "you are using $$RUSTC_VERSION"
+//@ normalize-stderr: "you are using [0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z0-9.]+)?( \([^)]*\))?" -> "you are using $$RUSTC_VERSION"
```

Comment:
@cuviper Our regex syntax treats this as a literal match and not a wildcard, right?

Reply:
I sure hope so -- that's true of every regex engine I know!

But really, it must be, or else this would be gobbling the rest of the line and definitely fail UI expectations.

Reply:
Yeah, that's my expectation too. Just that the next capturing group is `( \([^)]*\))?` which is what made me wonder about details of how the regex is getting interpreted.